lib/matplotlib/backends/backend_agg.py

```diff
@@ -490,6 +490,12 @@ def print_to_buffer(self):
     # print_figure(), and the latter ensures that `self.figure.dpi` already
     # matches the dpi kwarg (if any).
 
+    def print_gif(self, filename_or_obj, *, metadata=None, pil_kwargs=None):
+        # savefig() has already applied savefig.facecolor; we now set it to
+        # white to make imsave() blend semi-transparent figures against an
+        # assumed white background.
```

Comment:
```suggestion
```

This comment is about the context manager that did not get moved over from `print_jpg`.